### PR TITLE
Migration tweaks

### PR DESCRIPTION
**Pull request by dgasmith**
## Description
Accomplishes the following:
- Moves migration scripts to their own folder.
- Adds a new keywords tests for geometry optimizations
- Moves testing database to `test_qcarchivedb` to be more explicit

## Status
- [ ] Changelog updated
- [x] Ready to go

### CODE REVIEW

File: devtools/old_migrations/mongo_to_SQL_v0.6.0_v0.7.0.py [hunk 1]
```diff
@@ -9,9 +9,142 @@
 from qcfractal.storage_sockets.me_models import (MoleculeORM, KeywordsORM, KVStoreORM, ResultORM,
                                                  ProcedureORM, TaskQueueORM, QueueManagerORM, UserORM)
 from qcfractal.storage_sockets.sql_models import (MoleculeMap, KeywordsMap, KVStoreMap, ResultMap,
-                                                  ProcedureMap, TaskQueueMap)
+                                                  ProcedureMap, TaskQueueMap, Base)
 from qcfractal.interface.models import (KeywordSet, ResultRecord, OptimizationRecord,
                                         TorsionDriveRecord, GridOptimizationRecord)
+from qcfractal.storage_sockets.sqlalchemy_socket import SQLAlchemySocket
+
+from qcfractal.storage_sockets.sql_models import Base
```
Comment (Lnaden) [inline devtools/old_migrations/mongo_to_SQL_v0.6.0_v0.7.0.py]: replicated import from line 12

File: qcfractal/queue/managers.py [hunk 2]
```diff
@@ -592,7 +596,9 @@ def test(self, n=1) -> bool:
                         failed_program = program
                         break
                 if failed_program not in fail_report:
-                    fail_report[failed_program] = f"On test {k}: {result.error}"
+                    fail_report[failed_program] = f"On test {k}:" \
+                                                  f"\nException Type: {result.error.error_type}" \
+                                                  f"\nException Message: {result.error.error_message}"
```
Comment (Lnaden) [inline qcfractal/queue/managers.py]: I did not see this here, I think I'll remove this in my own PR since this should be better

Comment (dgasmith) [reply qcfractal/queue/managers.py]: I need to rebase anyway, wouldn't worry about it.

**Comment by Lnaden** [comment]
Looks like there is a file clash which needs to be resolved first

**Pull request merged**